### PR TITLE
test(lerobot_local): cover FeatureNormalizer torch-absent fallback + q10_q90 mode

### PR DESCRIPTION
**Pull request by cagataycali**
## What
`FeatureNormalizer` (`strands_robots/policies/lerobot_local/norm_stats.py`) is the normalization port wired into the LeRobot processor steps that run on the `norm_stats.json` fallback path. It is safety-critical: a normalizer that silently drops or mis-applies normalization sends raw-space values to the motors. Two parts of its documented contract had no test.

### 1. Optional-torch fallback (untested branch)
`_to_array` and `_restore_like` each wrap `import torch` in `except ImportError`, and the class docstring promises numpy-in/numpy-out when torch is absent. No test exercised the torch-less branch, so that behavior was unverified. The new test masks `sys.modules["torch"]` with `None` (making `import torch` raise `ImportError`) and asserts `normalize`/`unnormalize` still produce correct `np.ndarray` output.

### 2. `q10_q90` mode numerics (unvalidated)
Only `q01_q99` was validated against an analytic reference. `q10_q90` shares the quantile-scaling branch but selects the `q10`/`q90` stat keys; nothing pinned that key selection or its transform, so a regression swapping the keys would pass silently. The new test checks the forward map against the analytic form and an in-range round-trip.

## Tests
Both added to `TestFeatureNormalizer` in `tests/policies/lerobot_local/test_norm_stats_fallback.py`. Behavior-only assertions (outputs, not internals).

Result on the file's own run: `norm_stats.py` coverage **95% -> 97%** (12 -> 8 uncovered lines; the torch-absent `_to_array`/`_restore_like` branches are now covered). Full file suite: **53 -> 55 passed**.

```
before: norm_stats.py  242  12  95%  71-73, 194, 222, 235-238, 371-372, 442, 446, 452-453
after:  norm_stats.py  242   8  97%  194, 222, 371-372, 442, 446, 452-453
```

Gate: `ruff check` + `ruff format --check` + `mypy strands_robots tests tests_integ` clean; `MUJOCO_GL=egl` pytest of the file green. Test-only change; no production code touched.

### CODE REVIEW

**Review by yinsong1986 — APPROVED** [review]
## Summary

Test-only PR (+31/-0, one file) adding two regression tests to `TestFeatureNormalizer` in `tests/policies/lerobot_local/test_norm_stats_fallback.py`. The first pins `FeatureNormalizer`'s torch-absent fallback: it masks `sys.modules["torch"]` with `None` (making `import torch` raise `ImportError`) and asserts `normalize`/`unnormalize` still return correct `np.ndarray` output via the `_to_array`/`_restore_like` fallback branches. The second validates the `q10_q90` mode's forward map against the analytic quantile-scaling form and an in-range round-trip, pinning the `q10`/`q90` key selection that previously had no coverage. No production code is touched.

## What's good

- Behavior-only assertions (outputs, not internals), per AGENTS.md "Test behavior, not implementation".
- Verified locally: the torch-mask trick correctly raises `ImportError` and exercises the documented fallback; the `q10_q90` math matches production (`norm_stats.py:192`) exactly.
- Uses `monkeypatch.setitem` rather than mutating `sys.modules` directly, matching the project's teardown-safety convention.
- Tight scope, no host paths, no emojis, no secrets.
